scalableminds/webknossos
# Reload page when the next task is loaded which has a different script

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Create a task with a script which outputs something upon start.
- Switch from one task to another with "finish and get next task"
- The page should reload and the output should be visible after switching the task.

### Issues:
- contributes to https://github.com/scalableminds/webknossos/issues/5199

------
